src/tests/multihost/alltests/test_sssctl_analyzer.py

```diff
@@ -295,3 +301,35 @@ def test_analyze_parse_child_logs(self, multihost, backupsssdconf):
             assert re.findall(r"RID#[0-9]*] Received error code", stdout)
         else:
             pytest.fail(f"{user} sucessful to login")
+
+    def test_non_root_privileged(self, multihost, backupsssdconf, localusers):
```

Comment:
self is not used, please remove it from the parameters and add @staticmethod decorator instead.
backupsssdconf is probably unneeded as the cofiguration is not changed in the test.

Reply:
Entire test script is written with same format . so i am not going to break the format and i can see backupsssdconf is used every test case

Reply:
Done